test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Link.cshtml

```diff
@@ -46,6 +46,35 @@
           asp-fallback-test-property="visibility"
           asp-fallback-test-value="hidden" />
 
+    <!-- Fallback to static href with no asp-fallback-integrity-check attribute, default behavior is to keep integrity hash -->
+    <link href="~/styles/site.min.css?a=b&c=d" rel="stylesheet" data-extra="test" title='"the" title'
+          integrity="XY7YsMemPf8AGU4SIX9ED9eOjK1LOQWu2dmCNmh+pQc="
+          crossorigin="anonymous"
+          asp-fallback-href="~/styles/site.css?a=b&c=d"
+          asp-fallback-test-class="hidden"
+          asp-fallback-test-property="visibility"
+          asp-fallback-test-value="hidden" />
+
+    <!-- Fallback to static href with asp-fallback-integrity-check set to true, default behavior is to keep integrity hash -->
+    <link href="~/styles/site.min.css?a=b&c=d" rel="stylesheet" data-extra="test" title='"the" title'
+          integrity="XY7YsMemPf8AGU4SIX9ED9eOjK1LOQWu2dmCNmh+pQc="
+          crossorigin="anonymous"
+          asp-fallback-href="~/styles/site.css?a=b&c=d"
+          asp-fallback-test-class="hidden"
+          asp-fallback-test-property="visibility"
+          asp-fallback-test-value="hidden"
+          asp-fallback-integrity-check="true" />
```

Comment:
@mkArtakMSFT @DamianEdwards, in @jmhmaine's original PR he was kind enough to also fix up our `LinkTagHelper`. I went ahead and fixed that piece up and followed the same conventions as the `ScriptTagHelper`. Prior to merging this in I wanted to run that decision by you two seeing that it differs from the original plan to just add this attribute to the `ScriptTagHelper`.